test/arrays/reallocation/reallocInPlace-ml.numlocales

```diff
@@ -0,0 +1 @@
+2
```

Comment:
It seems like there's nothing about this test that wouldn't work on a single locale is that right?

Reply:
We don't register memory for -nl 1, so we wouldn't test the regMem interface with -nl1

Reply:
We wouldn't test that interface, but that doesn't make the test invalid for the 1-locale case, nor does it seem like the test output would vary on single-locale runs.  Or am I missing something subtle?

Reply:
No, you're not missing anything subtle, but in the interest of pushing on the comm layer interface specifically, this test's array is much larger and since on a single locale it wouldn't test anything not already covered by that first test you added, I decided to skip it for comm=none.